plugins/mapper-annotated-text/build.gradle

```diff
@@ -19,3 +19,9 @@ restResources {
     include '_common', 'indices', 'index', 'search'
   }
 }
+
+tasks.named('splitPackagesAudit').configure {
+    // TODO: rename this package to be annotated text specific
```

Comment:
Should we use `TODO:` in the other places as well?

Reply:
All of these are implicitly todo. I don't think it's necessary, since we don't use TODOs consistently or for working down todo items.